core/src/plugins/filed/python/vmware/BareosFdPluginVMware.py

```diff
@@ -757,6 +859,39 @@ def restore_object_data(self, ROP):
         ] = self.vadp.restore_objects_by_diskpath[ro_filename][-1]
         return bareosfd.bRC_OK
 
+    def create_restoreobject_savepacket(self, savepkt, virtual_filename, object_data):
+        """
+        Create a savepacket for a restore object
+        """
+        # create a stat packet for a restore object
+        statp = bareosfd.StatPacket()
+        savepkt.statp = statp
+        # see src/filed/backup.c how this is done in c
```

Comment:
```suggestion
        # see src/filed/backup.cc how this is done in c++
```

Reply:
Let's remove this comment, it has never been very helpful. As soon as we have a description of restore objects in the developer guide, we can add a comment with a link here.